### PR TITLE
Update readme and vignette

### PR DESCRIPTION
**Pull request by thevaachandereng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,11 +15,37 @@ LPWC also supports irregular time intervals between time points collected in bio
 
 Installation
 ------------
-The first major release of LPWC will be added to CRAN.
-Until then, the easiest way to install LPWC is as follows:
+Prior to analyzing your data, the R package needs to be installed.
+
+The easiest way to install LPWC is through R:
```
Comment (agitter) [inline README.md]: Should we say `through CRAN` instead of `through R`?

File: README.md [hunk 2]
```diff
@@ -15,11 +15,37 @@ LPWC also supports irregular time intervals between time points collected in bio
...
+
 ``` r
+install.packages("LPWC")
+``
```
Comment (agitter) [inline README.md]: This should be the triple ```

File: README.md [hunk 3]
```diff
@@ -15,11 +15,37 @@ LPWC also supports irregular time intervals between time points collected in bio
...
+There are other additional ways to download LPWC.
+The first option is most useful if using a previous versions of LPWC
+(which can be found at https://github.com/gitter-lab/LPWC/releases).
+
+``` r 
+devtools::install_github("gitter-lab/LPWC@vx.xx.x")
+# OR 
+library(devtools)
+install_version("LPWC", version = "x.x.x", repos = "http://cran.us.r-project.org")
+``
```
Comment (agitter) [inline README.md]: Triple ```

File: README.md [hunk 4]
```diff
@@ -15,11 +15,37 @@ LPWC also supports irregular time intervals between time points collected in bio
...
+Prior to analyzing your data, the R package needs to be installed.
+
+The easiest way to install LPWC is through R:
+
 ``` r
+install.packages("LPWC")
+``
+
+There are other additional ways to download LPWC.
+The first option is most useful if using a previous versions of LPWC
```
Comment (agitter) [inline README.md]: `version`

Does this mean that the first option is best if they already have a previous version of LPWC and want to update to the current version?

File: README.md [hunk 5]
```diff
@@ -15,11 +15,37 @@ LPWC also supports irregular time intervals between time points collected in bio
...
+install_version("LPWC", version = "x.x.x", repos = "http://cran.us.r-project.org")
+``
+
+The third option is to download through GitHub. 
+
+``` r
+library(devtools)
 devtools::install_github("gitter-lab/LPWC")
-```
+``
```
Comment (agitter) [inline README.md]: Triple ```

File: README.md [hunk 6]
```diff
@@ -15,11 +15,37 @@ LPWC also supports irregular time intervals between time points collected in bio
...
+
+After successful installation, the package must be loaded into the working space:
+
+``` r 
+library(LPWC)
+``
```
Comment (agitter) [inline README.md]: Triple ```

File: vignettes/LPWC.Rmd [hunk 7]
```diff
@@ -47,13 +47,31 @@ just reply that the question should be posted to the
...
 
-Because the package is not yet published on Bioconductor or CRAN, the best way to download the R package is through GitHub.
+The easiest way to install LPWC is through R:
+```{r, eval = FALSE}
+install.packages("LPWC")
+```
+
+
+There are other additional ways to download LPWC.
+The first option is most useful if using a previous versions of LPWC
```
Comment (agitter) [inline vignettes/LPWC.Rmd]: `version`, and same question as above in the readme.

File: README.md (hunk 1 above)
Comment (thevaachandereng) [reply README.md]: Let's do CRAN

File: README.md [hunk 8]
```diff
@@ -15,15 +15,40 @@ LPWC also supports irregular time intervals between time points collected in bio
...
+
 ``` r
+install.packages("LPWC")
+```
+
+There are other additional ways to download LPWC.
+The first option is most useful if want to download a specific version of LPWC
+(which can be found at https://github.com/gitter-lab/LPWC/releases).
+``` r 
+devtools::install_github("gitter-lab/LPWC@vx.xx.x")
```
Comment (agitter) [inline README.md]: Do we also need `library(devtools)` before this command, or is that not necessary?

Comment (thevaachandereng) [reply README.md]: we dont need library(devtools) here

**Pull request merged**